tests/framework/installer/installer.go

```diff
@@ -85,10 +86,19 @@ func init() {
 		createBaseTestDir = false
 		baseTestDir = "/data"
 	}
+	if RookTestOpenshift {
+		baseTestDir = "/var/lib/rook"
+		fmt.Sprintf("baseTestDir set to %s", baseTestDir)
+	}
 }
 
 func SystemNamespace(namespace string) string {
-	return fmt.Sprintf("%s-system", namespace)
+	if RookTestOpenshift {
```

Comment:
I remember it didn't work for me when it was in two namespaces even if I changed system:serviceaccount: NAMESPACE in SCC definition.  Do you know what can cause the issues on openshift? Or do you think of any issue if we will use just one NS when testing on OCP @travisn ?

Reply:
- One issue with OpenShift can be that working across namespaces might be disabled by default. #2723 has the documentation for working with that.
- Working with a single namespace is a good place to start. Working with multiple namespaces just ensures that scenario is working. Ideally we would have a mix of tests using both options.